src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -6,32 +6,41 @@
 package frc.robot.subsystems;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-import com.fasterxml.jackson.databind.deser.impl.CreatorCandidate.Param;
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 import edu.wpi.first.wpilibj.DigitalInput;
 import frc.robot.Parameters;
 
-
-import edu.wpi.first.wpilibj2.command.SubsystemBase;
-
 public class Intake extends SubsystemBase {
   /** Creates a new Intake. */
   public Intake() {}
 
-  //create intake motor!!!!!!
-  CANSparkMax intakeMotorBottom = new CANSparkMax(Parameters.driveTrain.can.INTAKEMOTORBOTTOM_ID, MotorType.kBrushless);
-  CANSparkMax intakeMotorTop = new CANSparkMax(Parameters.driveTrain.can.INTAKEMOTORTOP_ID, MotorType.kBrushless);
+  //create intake motors!!!!!!
+  CANSparkMax intakeMotorBottom = new CANSparkMax(Parameters.intake.can.INTAKE_MOTOR_BOTTOM_ID, MotorType.kBrushless);
+  CANSparkMax intakeMotorTop = new CANSparkMax(Parameters.intake.can.INTAKE_MOTOR_TOP_ID, MotorType.kBrushless);
 
 
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
   }
 
-  public void turnOn(){
-    intakeMotorTop.set(Parameters.intake.INTAKEMOTORTOP_SPEED);
-    intakeMotorBottom.set(Parameters.intake.INTAKEMOTORBOTTOM_SPEED);
+  //turns on the intake
+  public void setForward(){
```

Comment:
Pass a double in as a parameter and we can then use constant when we call this method.